src/push-notifications.js

```diff
@@ -206,4 +238,24 @@ function urlBase64ToUInt8Array(base64String) {
   return Uint8Array.from([...rawData].map(char => char.charCodeAt(0)));
 }
 
+/**
+ * Temporary whilst we only support Chrome in the Beta release.
+ * Modified from https://stackoverflow.com/questions/4565112
+ */
+function isChrome() {
+  const isChromium = window.chrome;
+  const winNav = window.navigator;
+  const vendorName = winNav.vendor;
+  const isOpera = typeof window.opr !== 'undefined';
+  const isIEedge = winNav.userAgent.indexOf('Edge') > -1;
+
+  return (
+    isChromium !== null &&
+    typeof isChromium !== 'undefined' &&
+    vendorName === 'Google Inc.' &&
+    isOpera === false &&
```

Comment:
We could just allow Opera, it works for opera as well. And that would be a ~2% market share win :)